### ISSUE TITLE
ability to set image title

### ISSUE THREAD

**Issue by b3arJS** [octref/polacode#56]
I suggest that an ability to set image title with an input field in VsCode could be helpful.

**Comment by filvi** [comment]
Yeah it would be really nice to have although as a workaround you can write a "stylish" (airquotes) comment at the very beginning of the code
